lib/python/venv.rb

```diff
@@ -0,0 +1,71 @@
+require 'pty'
+
```

Comment:
This file is just a rename of `/lib/cdo/python_venv.rb`, but the indentation change to make it Python::Venv instead of PythonVenv caused git to think its a new file.